<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelChooseWidget</class>
 <widget class="QWidget" name="ModelChooseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_model_widget_up" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <property name="verticalSpacing">
             <number>10</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_leave_model">
              <property name="styleSheet">
               <string notr="true">color: rgb(142, 227, 235);
font: 14pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>离家模式:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="mySwitchButton" name="pbn_leve_home">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_go_model">
              <property name="styleSheet">
               <string notr="true">color: rgb(142, 227, 235);
font: 14pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>回家模式:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="mySwitchButton" name="pbn_go_home">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_model_widget_down" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_led_note">
              <property name="styleSheet">
               <string notr="true">color: rgb(142, 227, 235);
font: 14pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>一键关关灯:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="mySwitchButton" name="pbn_key_led">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_text">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Abyssinica SIL&quot;;
color: rgb(142, 227, 235);</string>
         </property>
         <property name="text">
          <string>模式说明

【离家模式】
    切换到离家模式后系统会自动进行控制,     
无需手动控制
【回家模式】
        切换到回家模式后可以手动控制系统
【一键控制】
        一键控制可以控制家中所以灯的两灭</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mySwitchButton</class>
   <extends>QPushButton</extends>
   <header>myswitchbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
